src/pages/GapsPage.tsx

```diff
@@ -43,11 +43,37 @@ const Cell = styled.div`
 const TitleCell = styled(Cell)`
   font-weight: bold;
 `
+const TitleCellPrecentage = styled(TitleCell)`
+  width: 240px;
+`
+function displayGapsPercentage(gapsPrecentage: number) {
+  switch (true) {
+    case gapsPrecentage === 0:
+      return (
+        <TitleCellPrecentage style={{ color: '#097d09' }}>
```

Comment:
colors can be moved to constants to improve readability and consistency

Reply:
No problem, I can also add css classes for these colors, so we could use them in other elements if needed. Name of the classes: "great-result" (for green), "average-result" (for yellow), "terrible-result" (for red)

Reply:
even better!